servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -172,43 +172,33 @@ def query_jobs(body, **kwargs):
     headers = kwargs.get('auth_headers')
     query = QueryJobsRequest.from_dict(body)
     query_page_size = query.page_size or _DEFAULT_PAGE_SIZE
-    # Request more than query.page_size from cromwell since subworkflows will get filtered out
-    page_size = query_page_size * 2
     total_results = get_total_results(query, auth, headers)
+    if query.page_token is None:
```

Comment:
If the pages are in the proper order now, then you shouldn't need to get `total_results` first anymore; this was only done previously in order to find the "last page", then iterate backwards.

Reply:
`total_results` is also being used to figure out `last_page`, and that is used to figure out if a `next_page_token` should be returned

Reply:
But you should be able to get this value at line 183 without making this extra XHR, right?

Reply:
oh, right – because Cromwell will actually be returning the correct value 🙂

Reply:
fixed